simulation_configs/sample_config.ini

```diff
@@ -0,0 +1,36 @@
+[SimulationOne]
```

Comment:
Should there be an example that includes the `Frequency` key?

Reply:
I feel like it might give the false sense that you can run a simulation with a different point frequency when in reality you are always running the file as if it's 10 Hz, frequency is pretty much only useful if you want to make a route independently of the simulator I think. Maybe it shouldn't even be a configurable parameter for the simulation config

Reply:
I agree. If a user shouldn't be able to configure it, it shouldn't be exposed.